src/main.rs

```diff
@@ -107,6 +110,25 @@ fn main() {
         println!("{}", text);
     }
 }
+fn spawn_watch_shell(failed_exercise_hint: &Arc<Mutex<Option<String>>>) {
+    let failed_exercise_hint = Arc::clone(failed_exercise_hint);
+    println!("type 'hint' to get help");
```

Comment:
Please use proper capitalization here, e.g. `Type 'hint' to get help`

Reply:
good catch, I have fixed this and other issue.